src/api.toit

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2025 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be found
+// in the LICENSE file.
+
+import .client as client
+import system.services show ServiceClient ServiceSelector
+
+interface ObservableStateService:
+  listen handle/int -> none
+  static LISTEN-INDEX ::= 100_000_000  // Avoid conflicts with other methods.
```

Comment:
An alternative would be to take the index in the constructor.
But I'm fine this way.